libs/wingc/src/jsify.rs

```diff
@@ -371,6 +371,13 @@ impl<'a> JSifier<'a> {
 					BinaryOperator::Sub => "-",
 					BinaryOperator::Mul => "*",
 					BinaryOperator::Div => "/",
+					BinaryOperator::FloorDiv => {
+						return format!(
+							"Math.floor({} / {})",
```

Comment:
hmmm... no special ideas how to simplify that @flyingImer. @yoav-steinberg - do you have any idea?

Reply:
LMK

Will keep it in back of my mind as digging more into the art part of coding rust

Reply:
Maybe do a `let js_left = self.jsify_expression(left, phase);` and `let js_right = self.jsify_expression(right, phase);` before entering this `match`, to de-dup the call to `jsify_expression`. But other than that this seems fine.